code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -553,6 +553,16 @@
 	for(var/i = 1; i <= 5; i++)
 		new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/mousecube(src)
 
+/obj/item/weapon/storage/box/monkeycubes/spacecarpcube
+	name = "space carp baby cube box"
+	desc = "Drymate brand space carp baby cubes, shipped from F1SH-1NG. Just add water!"
+	icon_state = "spacecarpcubebox"
+
+/obj/item/weapon/storage/box/monkeycubes/spacecarpcube/New()
+	..()
```

Comment:
you do realize this means the box will get filled up with money cubes before it tries to add space carp cubes right?

Reply:
That's how all the other mobcube boxes are coded, I just repeated the previous lines, quite literally copypasted the mousecube code and replaced the lines accordingly.

Reply:
wow, it just werks because of some really offensively bad code by N3X15 of all people

*really*?

Reply:
apparently